en/patterns/grid.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Grid - Design System Pattern
+_description: The Grid Pattern Symbol represents a full-blown Grid Component.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid
+
+Use the Grid Pattern symbol as an alternative starting point when designing a tabular data layout. It provides a full-blown contentwise and stylewise Grid Component that comes with all the Grid functionality, provided in the Components Library such as:
+
+- Sorting
+- Filtering
+- Summaries
+- Paging
+- Editing
+- Column Moving
+- Column Resizing
+- Column Hiding
+- Display Density
+- Excel Style Filtering
+- GroupBy
+- Pinning
+- Row Selection
+- Row Editing
+- Active Cell
+- Export
+- Toolbar
+
+
+
+Grid Pattern symbol comes with six pre-made combinations of Grid features:
+- Grid Pattern with Summaries, Column Moving and Paging
+- Grid Pattern with Editing and Filter Row
+- Grid Pattern with Excel Style Filter, Sorting and Display Density
+- Grid Pattern with Group By, Column Resizing and Summaries
+- Grid Pattern with Pinning, Row Selection and Active Cell
+- Grid Pattern with Toolbar, Export, Row Editing and Column Hiding
+
+
+Below is illustrated the Grid Pattern with Excel Style Filter, Sorting and Display Density
+<img class="responsive-img" src="../images/grid.png" srcset="../images/grid@2x.png 2x" />
```

Comment:
Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.